### PR TITLE
Tweak command line scripts

### PR DESCRIPTION
**Pull request by willthefrog**
- enable fine grained control of command flags parsing
- switch to attribute style of accessing config options

### CODE REVIEW

File: PaddleCV/object_detection/ppdet/utils/cli.py [hunk 1]
```diff
@@ -40,50 +39,40 @@ def with_code(self, code, message):
...
-        help="The checkpoint path for resuming training.")
-    parser.add_argument(
-        "--eval",
-        action='store_true',
-        default=False,
-        help="Whether perform evaluation in train")
-    parser.add_argument(
-        "-o", "--opt", nargs=REMAINDER, help="set configuration options")
-    args = parser.parse_args()
+def parse_opt(opts):
```
Comment (heavengate) [inline PaddleCV/object_detection/ppdet/utils/cli.py]: 感觉这个作为ArgsParser的成员函数好点？

Comment (willthefrog) [reply PaddleCV/object_detection/ppdet/utils/cli.py]: done

**Review by heavengate — COMMENTED** [review]
LGTM

**Review by heavengate — APPROVED** [review]
LGTM

**Comment by willthefrog** [comment]
merged into #2517

**Pull request closed without merging**